cypress.config.ts

```diff
@@ -18,6 +18,7 @@ export default defineConfig({
         jira_atlassian_cloud_token: "",
         jira_atlassian_cloud_url: "",
         jira_atlassian_cloud_project: "Test",
+        jira_stage_datacenter_project_id: 12339124,
```

Comment:
Project Name doesn't work ?

Reply:
It doesn't because in the stage API we don't have access to list all project nor to search projects by name.